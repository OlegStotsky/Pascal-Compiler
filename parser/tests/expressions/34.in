var
	a : char;
	b : char;

begin
	a := 'a';
	b := 'b';
	a := a + b;
end;