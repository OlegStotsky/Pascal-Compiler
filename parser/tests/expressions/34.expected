BEGIN SYMTABLE
 VAR
  NAME : a
  TYPE : char
 VAR
  NAME : b
  TYPE : char
 TYPE : boolean
 TYPE : char
 TYPE : integer
 TYPE : float
END SYMTABLE
BEGIN STMT BLOCK
 BEGIN STMT ASSIGN
   a
  :=
   a
 END STMT ASSIGN
 BEGIN STMT ASSIGN
   b
  :=
   b
 END STMT ASSIGN
 BEGIN STMT ASSIGN
   a
  :=
    a
   +
    b
 END STMT ASSIGN
END STMT BLOCK
